### ISSUE TITLE
Minio Console can not be served on subpath using the latest Minio version

### ISSUE THREAD

**Issue by sidohaakma** [minio/minio#14285]
When you are try to serve Minio Console on a subpath it won't adhere the CONSOLE_SUBPATH environment variable. Whci makes it impossible to server Minio Console on a subpath. We are using docker-compose to deploy with NGINX as a reverse proxy to expose the service.

This is the current configuration:

`docker-compose` configuration

```yaml
version: "3"

services:
  minio:
    image: quay.io/minio/minio:RELEASE.2022-02-07T08-17-33Z
    ports:
      - 9001:9001
      - 9000:9000
    environment:
      CONSOLE_SUBPATH: /console/
      MINIO_ROOT_USER: admin
      MINIO_ROOT_PASSWORD: verysecretpassword
      .....
```

NGINX configuration

```conf
...
   location /console/ {
        rewrite ^/console/(.*)$ $1 break;
        proxy_pass http://localhost:9001;
   }
...
```

## Expected Behavior
I would expect that the `<base href='/'/>` would be set to `<base href='/console'/>` in the index.html. Which allows the resources to be served properly.

## Current Behavior
The tag is set to `<base href='/'/>`. It seems that the environment variable is not adhered at the moment within the minio binary. It looks like the correct version of the minio console module is included: https://github.com/minio/minio/blob/RELEASE.2022-02-07T08-17-33Z/go.mod. 0.14.3. 

## Possible Solution
Maybe I overlooked something in the configuration or it is not supported yet in the minio deployment itself?

## Steps to Reproduce (for bugs)
1. Create a docker-compose.yml with the cofiguration above
2. Deploy an NGINX which proxies the console with the configuration above
3. Check http://localhost/console/

## Context
We are deploying a federated analysis platform remotely on servers accross Europe using Minio as a data storage. It is quite a hassle to get these deployments in place, because of the responsibility of IT-departments are scattered over different teams. Therefor acquiring additional domain entries is quite a hassle.

## Regression
I think not. This is due to the upgrade to the new Minio. We upgraded from a Minio which was serving the console the old way on the root on the same port as the api.

## Your Environment
* Version used (`minio --version`): quay.io/minio/minio:RELEASE.2022-02-07T08-17-33Z
* Server setup and configuration: docker-compose on the OS with NGINX on the OS without docker as a reverse proxy
* Operating System and version (`uname -a`): Ubuntu

**Comment by sidohaakma** [comment]
I can attach the full docker-compose and nginx configuration if necessary.

**Comment by harshavardhana** [comment]
Sub-path is not supported - we will try to support subpath in future.

**Comment by federicober** [comment]
Just wanted to say that this is a feature that would be super useful!

**Comment by harshavardhana** [comment]
This will be fixed in #14761
